### DISCUSSION TITLE
JsResourceHandler

### DISCUSSION THREAD

**Discussion by paulmcgann** [valdisiljuconoks/LocalizationProvider#267] [Q&A] [answered]
Hi,

Currently we are running through a cms12 / commerce 14 upgrade and was just wondering if the JsResourceHandler had a corresponding package to v7?

The current package is 6.5.2 however this appears incompatible with the v7.

Thanks

Paul

**Comment by valdisiljuconoks** [comment, marked as answer]
Hi,

Sorry for super delayed answer (this thread have slipped through my inbox). Client-side translation is now part of the standard package (`DbLocalizationProvider.AspNetCore`) and more info could be found [here](https://github.com/valdisiljuconoks/localization-provider-core/blob/master/docs/client-side-provider-netcore.md) on how to get started.

**Reply by paulmcgann** [reply]
Hi,

Thanks for the updated information.

Makes perfect sense now.

Appreciate all the effort you have put into this for the community, keep up the good work!

Paul
